builder/solver.go

```diff
@@ -166,6 +166,8 @@ func newCacheExportOpt(ref string, max bool) client.CacheOptionsEntry {
 	registryCacheOptAttrs["ref"] = ref
 	if max {
 		registryCacheOptAttrs["mode"] = "max"
+		registryCacheOptAttrs["image-manifest"] = "true"
```

Comment:
once we handle parsing the image string, I think it's better to assign the attr values outside of this `if` statement.

Reply:
Got it.